device/linux/wifi/wifi.cpp

```diff
@@ -32,7 +32,7 @@ wifi_err_t wifi_disconnect() {
   return RES_OK;
 }
 
-bool wifi_is_disconnected() {
+bool wifi_is_connected() {
```

Comment:
Default true?

Reply:
this is a stub impl for my laptop. so yes wifi is connected